.github/workflows/tools.yml

```diff
@@ -174,14 +173,6 @@ jobs:
               cat temp-output
               tail -n1 temp-output | grep "NEW_VERSION=" >> "$GITHUB_ENV" || true
               rm temp-output
-          - id: libuv
```

Comment:
Can we just comment this out instead of removing, I assume we will want to add it back at some point?

Reply:
The other question is if we could have the update apply a patch required to re-apply the changes?

Reply:
I don't know, technically we could, but we'll need to review it carefully all libuv automatic updates + resolve conflicts.